.circleci/config.yml

```diff
@@ -4,7 +4,7 @@ version: 2.1
 
 defaults: &defaults
   macos:
-    xcode: '11.4.1'
+    xcode: '12.0.0'
```

Comment:
Seems fine, but if circleci supports 12.0.1 (since it was released 2d ago, we should probably use that? )

Reply:
Circle CI [doesn't support 12.0.1 yet](https://circleci.com/docs/2.0/testing-ios/)